main.py

```diff
@@ -1,10 +1,66 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def load_data_from_json() -> dict:
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    return players_data
+
+
 def main() -> None:
-    pass
+    players_data = load_data_from_json()
+
+    for nickname, player_data in players_data.items():
+        race_data = player_data.get("race")
+        guild_data = player_data.get("guild")
```

Comment:
Ensure that 'race' and 'guild' keys are present in 'player_data'. Consider using a default value or handling the case where these keys might be missing to prevent KeyError exceptions.